UKHomeOffice/cerberus-service
# COP-10436: Add test to verify Passenger Details in Task overview page

## Description
Add test to verify Passenger Details in Task overview page

https://support.cop.homeoffice.gov.uk/browse/COP-10436
## To Test
npm run cypress:test:local -- --spec cypress/integration/airpax/airpax-tasks-details.spec.js

Should verify Passenger details of an AirPax task on task details page
## Developer Checklist

\* Required
